readHR.py

```diff
@@ -0,0 +1,124 @@
+class EcgReader:
+
+    def __init__(self, file):
+        """
+
+        :param file: Input file for ECG reading and calculations
+
+        """
+        self.file = file
+        self.csv_check = True
+        self.data_type = True
+        self.time = []
+        self.voltage = []
+        self.peak_vector = []
+        self.filecheck()
+        if self.csv_check:
+            self.readfile()
+            self.datacheck()
+        if self.data_type and self.csv_check:
+            self.findrange()
+
+    def filecheck(self):
+        """
+            This method takes in ECG data from a CSV file inputted by the
+            user, and checks that it is the correct file format.
+
+            :return: If the file is not a CSV, the function will raise
+            "Error: File is not a .csv". For CSV files, the initialization
+            will move on to the next step.
+
+        """
+        if self.file.endswith('.csv'):
+            self.csv_check = True
+        else:
+            self.csv_check = False
+            self.time = None
+            self.voltage = None
+            print("Error: File is not a .csv")
```

Comment:
you may want to return something to the programmer who is using your class here (probably a True/False) to indicate that the file did or did not checkout. You could consider throwing an exception that the downstream caller/user could catch (you could catch it in your init as well)

Reply:
would this be something returned only if you run the method separately?

Reply:
nah, you can just always return and the caller can choose if they want to grab it or not.